src/app/services/baw-api/sites.service.ts

```diff
@@ -38,7 +38,11 @@ export class SitesService extends StandardApi<Site, [IdOr<Project>]> {
     return this.apiList(endpoint(project, Empty, Empty));
   }
   filter(filters: Filters, project: IdOr<Project>): Observable<Site[]> {
-    return this.apiFilter(endpoint(project, Empty, Filter), filters);
+    return filterMock(
+      filters,
+      (index) => new Site({ id: index, name: "PLACEHOLDER" })
+    );
+    // return this.apiFilter(endpoint(project, Empty, Filter), filters);
```

Comment:
Why are we committing this change?

Reply:
Sites filter is broken I'm fairly certain.

Reply:
Add in a code comment then and link to relevant baw-server issue

Reply:
Baw issue: https://github.com/QutEcoacoustics/baw-server/issues/437